components/pd_client/src/tso.rs

```diff
@@ -54,24 +60,22 @@ pub struct TimestampOracle {
 impl TimestampOracle {
     pub(crate) fn new(
         cluster_id: u64,
-        pd_client: &PdClient,
-        call_option: CallOption,
+        pd_client: &PDClient<Channel>,
+        spawn_handle: tokio::runtime::Handle,
```

Comment:
This `spawn_handle` is not used?

Reply:
Yes, this is a redundant parameter that should be removed. Originally, I want to share the global tokio runtime to do the tso background task, but later changed to use a separate thread to keep it the same with the current implementation.

Reply:
Let's clean this up. I think we can minimize the parameter propagation a lot from it.

Reply:
Done. But unfortunately, its impact is very small. Because many tonic spawned future task must be run within a tokio runtime(maybe only some of them need not all), so we still need to pass or store the Handle in various places.